packages/x-charts/src/ChartContainer/ChartContainer.test.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { createRenderer, describeConformance } from '@mui/internal-test-utils';
+import { ChartContainer } from './ChartContainer';
+
+describe('<ResponsiveChartContainer />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<ChartContainer height={100} width={100} series={[]} />, () => ({
+    classes: {} as any,
+    inheritComponent: 'svg',
+    render,
+    muiName: 'MuiChartContainer',
+    testComponentPropWith: 'div',
+    refInstanceof: window.SVGSVGElement,
+    only: ['mergeClassName', 'propsSpread', 'refForwarding', 'reactTestRenderer', 'rootClass'],
```

Comment:
Might be better to use the `skip` option instead of the `only`

It took me sometime to hunderstand why the API generated had some `"themeDefaultProps": true,` even if we do not implement them.

I've open an issue for it https://github.com/mui/material-ui/issues/42832

Reply:
The equivalent with skip would be
```
skip: [
'componentProp',
'componentsProp',
'slotPropsProp',
'slotPropsCallback',
'slotsProp',
'themeDefaultProps',
'themeStyleOverrides',
'themeVariants',
'themeCustomPalette']
```